src/content/learn/manipulating-the-dom-with-refs.md

```diff
@@ -325,13 +342,13 @@ In this example, `itemsRef` doesn't hold a single DOM node. Instead, it holds a
   key={cat.id}
   ref={node => {
     const map = getMap();
-    if (node) {
-      // Add to the Map
-      map.set(cat.id, node);
-    } else {
-      // Remove from the Map
-      map.delete(cat.id);
-    }
+    // Add to the map
```

Comment:
This doesn't indicate canary? I also don't understand why we'd need to add/remove cats?

Reply:
Ah thanks I missed this spot. Updating to a Canary block. 

For the add/remove, this was to make it more obvious how/when the cleanups are fired if you want to extend the example with logs or other behavior. Was helpful when putting it together but I'll update to simplify here.